monkey/infection_monkey/network/HostFinger.py

```diff
@@ -0,0 +1,23 @@
+from abc import ABCMeta, abstractproperty, abstractmethod
+
+from infection_monkey.config import WormConfiguration
+
+
+class HostFinger(metaclass=ABCMeta):
+    @property
+    @abstractmethod
+    def _SCANNED_SERVICE(self):
+        pass
+
+    def init_service(self, services, service_key, port):
```

Comment:
_I realize this is just moved code and not something you've introduced._
This is less of "init_service" and more of "add_service_to_services". Output parameters are confusing. That got me to wondering: why this is the responsibility of `HostFinger.init_service(host.services, X, Y)` and not `host.add_service(X, Y)` or `host.add_service(this.compose_service_dict())` (host is `VictimHost` and `this` is HostFinger).

I think that would make much more sense. All usages of this function include host.services as an output parameter, clean code says this is not the best idea (see `Output Arguments` page 45).

Reply:
Why should I change this in this PR?